### PR TITLE
Improve logs: Extra payload and open telemetry integration

### PR DESCRIPTION
**Pull request by ilbambino**
Export to GCP in the structured format all the extra fields added to logrus. To do so, removed the `logEntry` struct and adding everything to a map to be able to serialise it in the correct format.

Span and Trace ID are added instead from the OpenTelemetry span. 

Tests simplified and adapted to new format

### CODE REVIEW

File: formatter_test.go [hunk 1]
```diff
@@ -1,134 +1,80 @@
...
-
-			assert.Equal(t, row.input, testEntry)
-		})
-	}
-}
-
 func TestFormat(t *testing.T) {
 	logger := logrus.New()
-
+	logger.Debug("test")
```
Comment (seungbemi) [inline formatter_test.go]: debug?

File: formatter.go [hunk 2]
```diff
@@ -70,21 +69,31 @@ func (f *Formatter) Format(entry *logrus.Entry) ([]byte, error) {
 		severity = value
 	}
 
-	newEntry := logEntry{
-		Message:  entry.Message,
-		Severity: severity.String(),
-		Labels:   map[string]string{},
-	}
+	mapEntry := map[string]interface{}{}
+	mapEntry["severity"] = severity.String()
```
Comment (seungbemi) [inline formatter.go]: shall we use const for all the keys for more visibilities?

**Review by seungbemi — APPROVED** [review]
LGTM

File: formatter_test.go [hunk 3]
```diff
@@ -1,134 +1,80 @@
...
+			result, err := formatter.Format(row.input)
+			require.Nil(t, err)
+			assert.Equal(t, row.output, result)
+		})
+	}
+}
+
+func TestFormatWithLabels(t *testing.T) {
+	logger := logrus.New()
+	logger.Debug("test")
```
Comment (seungbemi) [inline formatter_test.go]: here as well

File: formatter_test.go (hunk 1 above)
Comment (ilbambino) [reply formatter_test.go]: leftovers. fixed

**Pull request merged**